css/style.css

```diff
@@ -43,6 +42,121 @@ body {
   margin: auto;
   height: 500px;
 }
+.logoOne {
+  margin-left: 100px;
+  font-size: 24px;
+  font-weight: 900;
+  color: white;
+}
+.message {
+  text-align: center;
+  font-weight: bold;
+  font-size: 24px;
+  color: white;
+  margin-top: 150px;
+}
+.centerButton { 
+  text-align: center;
+  margin-top: 50px;
+}
+.getIt {
+  background-color: #f9e42e;
+  font-size: 14px;
+  font-weight: 900;
+  color: #121212;
+  border-radius: 5px;
+  padding: 10px 20px;
+}
+.greyBackground {
+  width: 100%;
+  height: 280px;
```

Comment:
Rather than setting a fixed height, see if you can avoid setting height at all. If you feel you need some extra white space, use `min-height`. Remember, you won't to avoid setting a fixed height on anything that contains text. Otherwise if a user is on a narrower screen, the text can easily overflow the container.